Add support for Footer in Landscape mode (#89)
Wootric/WootricSDK-Android: CHANGELOG.md

## 2.17.0 (2020-07-28)

> - Add support for showing Footer section in also Landscape mode

Reviewer: ```suggestion
- Add support to show Footer section in Landscape mode. 
```

Author: Thanks for correcting me here, this sounds more right 👍
